.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,21 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
+      - run: codespell --skip="*/misspelling.json" || true  # --ignore-words-list=""
```

Comment:
As mentioned in your other PR:
> It would be great if we could omit from `codespell` files with prefix `./augly/tests/text_tests/*` -- in our text unit tests, we test the output of a function called `simulate_typos` which will have expected typos. I'll comment this on your other PR as well, as that is where this change likely needs to be made.

Reply:
codespell --ignore-words-list="tha" --skip="*/text_tests,*/misspelling.json"